default.n2r

```diff
@@ -39,11 +37,11 @@ $COPYFILES
 # If any binaries are included, symlink them to bindir here
 $BINSNIPPET
 
-%nodejs_symlink_deps
+$NODEJS_SYMLINK_DEPS
```

Comment:
To avoid the manual fix in the spec result file according to:
```
# FIXME
# Remove nodejs_symlink_deps if installing bundled module with NPM
```
If I understand well the use of `%nodejs_symlink_deps`, we don't want this line in case of bundled module. But I sure it could be managed better than my suggestion. It's only an idea which I've used to do easier my packages, like the use of 100 for Epoch value.